docs/03-tutorials/00-api-exposure/apix-10-expose-and-secure-workload-with-certificate.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Expose and secure a workload with a certificate
+---
+
+This tutorial shows how to expose and secure a workload with mutual authentication using Kyma's mutual TLS Gateway.
+
+## Prerequisites
+
+This tutorial is based on a sample HttpBin service deployment and a sample Function. To deploy or create one of those, follow the [Create a workload](./apix-01-create-workload.md) tutorial.
+
+Before you start, Set up [mTLS Gateway](../00-security/sec-03-setup-mtls-gateway.md) to allow mutual authentication in Kyma and make sure that you exported the [bundle certificates](../00-security/sec-03-setup-mtls-gateway#steps).
+
+Optionally you can take a look on tutorial [How to create own self-signed Client Root CA and Certificate](../00-security/sec-02-mtls-selfsign-client-certicate.md)
+
+## Authorize client with a certificate
+
+Follow the instructions in the tabs to further secure the mTLS service or Function. Create AuthorizationPolicy that checks if the client's Common Name in the certificate matches.
```

Comment:
```suggestion
The following instructions describe how to further secure the mTLS service or Function. 
>**NOTE:** Create AuthorizationPolicy to check if the client's common name in the certificate matches.
```
Can you add some more information? With what`the client's common name in the certificate matches`?